### ISSUE TITLE
obj loader vt not compatible for Quadrilateral meshs

### ISSUE THREAD

**Issue by jingyangcarl** [mikedh/trimesh#1256]
https://github.com/mikedh/trimesh/blob/5276d45ff50b3fd937e24c72f46a7ec3f0b0d777/trimesh/exchange/obj.py#L404

using the column as an identifier can only used or triangular mesh.

**Comment by mikedh** [comment]
Yeah that's not currently implemented, PR's welcome! The test model in `models/quadknot.obj` might be good for a unit test.
